tests/profiling/exporter/test_pprof.py

```diff
@@ -780,7 +780,10 @@ def test_pprof_exporter_libs(gan):
             {"kind": "standard library", "name": "platstdlib", "version": platform.python_version()},
         )
 
-    assert libs == expected_libs
+    for _ in libs:
+        expected_libs.remove(_)
+
+    assert not expected_libs
```

Comment:
`set` won't work because `dict`s are not hashable. Even converting the `dict`s to tuple of tuples is not good enough, because some of the values are `list`s, which are mutable 🙁

Reply:
😢 a comment would be nice.

Reply:
ah yep I'll add one. Also need to figure out how to fix another test now that there is no sorting during collection :S